src/core_plugins/kibana/public/visualize/index.js

```diff
@@ -24,7 +25,7 @@ uiRoutes
   requireDefaultIndex: true
 })
 .when('/visualize', {
-  redirectTo: '/visualize/step/1'
+  redirectTo: '/visualize/landing'
```

Comment:
Maybe we should pick a more descriptive route name. Maybe `/visualize/load` or `/visualize/index` or just `/visualize` since it's an index of the visualizations.